src/renderer/components/sidebar-file-tree.tsx

```diff
@@ -52,11 +52,13 @@ export class SidebarFileTree extends React.Component<
           presence !== EditorPresence.Hidden ? 'eye-open' : 'eye-off';
 
         return {
+          isSelected: currentEditorId === editorId,
           id: index,
           hasCaret: false,
           icon: 'document',
           label: (
             <ContextMenu2
+              onClick={() => this.toggleVisibility(editorId)}
```

Comment:
In keeping with @nornagon's comments from the issue, i agree that we should show and focus the editor if it's not visible. What this does, however, is makes it such that clicking once shows the editor, and another click hides it. I don't think that's intuitive behavior nor behavior users would want. We should _only_ show, never hide here.

Reply:
According to what you say:
1. If not visible, display and focus
2. if visible, what is the behaviour? Is it just focus?

Reply:
Yes. When clicked, ensure the clicked item is visible and has focus.